lib/plugins/leetcode.js

```diff
@@ -120,7 +120,7 @@ plugin.getCategoryProblems = function(category, cb) {
 plugin.getProblem = function(problem, cb) {
   log.debug('running leetcode.getProblem');
   const user = session.getUser();
-  if (problem.locked && !user.paid) return cb('failed to load locked problem!');
+  //if (problem.locked && !user.paid) return cb('failed to load locked problem!');
```

Comment:
for non-premium user, what's the error now if fetching a locked question?

Reply:
It will show "[ERROR] failed to load problem! [0]" when non-premium user try to access locked problem.

Reply:
Cool, this works for me